packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -312,9 +324,13 @@ function ActiveRequests({ votingAccount, votingGateway, snapshotContract }) {
       const currentVotes = await Promise.all(
         voteStatuses.map(async (voteStatus, index) => {
           if (voteStatus.committedValue) {
-            return JSON.parse(
-              await decryptMessage(decryptionKeys[account][currentRoundId].privateKey, voteStatus.committedValue)
-            );
+            try {
+              return JSON.parse(
+                await decryptMessage(decryptionKeys[account][currentRoundId].privateKey, voteStatus.committedValue)
+              );
+            } catch (err) {
+              return "";
```

Comment:
If we're going to catch this error, can we return a string value that indicates the error? I think this should be distinguishable from no vote in some way. Something like "ERROR: DECRYPTION FAILED"? Would be ideal to show that in the "value" field in the dapp. Another (slightly less ideal) solution could be to print out the error using console.error so we can diagnose the issue in prod. Otherwise the error gets swallowed and there's no way to find the cause.

Reply:
yes, this should log to console.error since theres no way to really notifiy the user in this app